### PR TITLE
192 update user profile ui

### PR DESCRIPTION
**Pull request by MatyashDen**

### CODE REVIEW

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt [hunk 1]
```diff
@@ -58,38 +61,53 @@ class UserActivity : AppCompatActivity() {
...
-        getString(R.string.cur_rating, rating.toString())
+        return str
     }
 
-    private fun User.buildName() = when {
+    private fun User.buildName() = colorTextByUserRank(when {
         firstName == null && lastName == null -> getString(R.string.none)
-        firstName == null -> lastName
-        lastName == null -> firstName
+        firstName == null -> lastName!!
```
Comment (yevhenii-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt:101]: `?.orEmpty()` or `?: ""` and the same for all new `!!`. Let's not risk crashing the app.

File: common/src/commonMain/sqldelight/io/xorum/codeforceswatcher/6.sqm [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE DbUser ADD COLUMN contribution INTEGER;
```
Comment (yevhenii-kanivets) [inline common/src/commonMain/sqldelight/io/xorum/codeforceswatcher/6.sqm:1]: Oups, will be needed to merged cautiously.

**Review by yevhenii-kanivets — CHANGES_REQUESTED** [review]
Doesn't compile on my side 😢 Missing arguments in `DatabaseQueries.insert`.

**Comment by MatyashDen** [comment]
@yevhenii-kanivets 
Oops, forgot to add all files to git

**Review by yevhenii-kanivets — APPROVED** [review]
**it happens! Good now 👍

**Pull request merged**